flathub.json

```diff
@@ -0,0 +1,6 @@
+{
+  "only-arches": [
```

Comment:
you don't need this these are the only arch supported today.

Reply:
In other words, arm64 is not supported?

Reply:
only `aarch64` and `x86_64` so its not needed to say they are the only one.

(`arm64` is `aarch64` in case it wasn't clear)